ui/src/app/components/app-router.spec.tsx

```diff
@@ -2,21 +2,12 @@ import '@testing-library/jest-dom';
 
 import * as React from 'react';
 import { MemoryRouter } from 'react-router';
-import { Redirect, Switch } from 'react-router-dom';
```

Comment:
Navigate is the [replacement](https://stackoverflow.com/questions/69961684/what-is-the-alternative-for-redirect-in-react-router-dom-v-6-0-0) for Redirect